index.html

```diff
@@ -61,31 +61,26 @@ <h3>pic-editor - We start from here ? </h3>
                 <h3>No! - We start from here </h3>
             </section>
             <section>
-                <h3>True value: business centric approach</h3>
-            </section>
-            <section>
-                <h3>We finally toggeter </h3>
+                <h3>We finally together</h3>
```

Comment:
Non mi torna in inglese.. Cosa volevi dire esattamente? Forse
```suggestion
                <h3>Us, finally together</h3>
```
?